subscriptions/models.py

```diff
@@ -45,12 +70,33 @@ def __str__(self) -> str:
         return self.codename
 
 
+class Feature(models.Model):
```

Comment:
Add `__str__`, otherwise 
![Screenshot_2022-10-24_16-03-01](https://user-images.githubusercontent.com/73276794/197531933-72473108-1ca1-4092-8821-bd0ddf19c64e.jpg)